main.go

```diff
@@ -324,76 +364,31 @@ For more information, please go to https://github.com/brancz/kube-rbac-proxy/iss
 	var gr run.Group
 	{
 		if cfg.secureListenAddress != "" {
-			srv := &http.Server{Handler: mux, TLSConfig: &tls.Config{}}
+			shutdownTimeout := time.Duration(60) * time.Second
+			stopCh := make(chan struct{})
+			var stoppedCh <-chan struct{}
```

Comment:
why is this and listenerStoppedCh scoped out here instead of inside the anonymous func?